packages/ppx_import/ppx_import.1.10.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "A syntax extension for importing declarations from interface files"
+maintainer: "whitequark <whitequark@whitequark.org>"
+authors: [ "whitequark <whitequark@whitequark.org>" ]
+homepage: "https://github.com/ocaml-ppx/ppx_import"
+doc: "https://ocaml-ppx.github.io/ppx_import/"
+license: "MIT"
+bug-reports: "https://github.com/ocaml-ppx/ppx_import/issues"
+dev-repo: "git+https://github.com/ocaml-ppx/ppx_import.git"
+tags: [ "syntax" ]
+
+depends: [
+  "ocaml"                   {>= "4.05.0" &  < "4.10.0"  } | ("ocaml" {>= "4.10.0"} "ppx_sexp_conv" {with-test & >= "v0.13.0"})
+  "dune"                    {              >= "1.11.0"  }
+  "ppxlib"                  {              >= "0.26.0"  }
+  "ounit"                   { with-test                 }
+  "ppx_deriving"            { with-test  & >= "4.2.1"   }
```

Comment:
I think this will make ppx_import `-with-test` uninstallable on OCaml 4.09 downwards.

Reply:
I’m not seeing it. How so? (also this was already there in the original formula)

Reply:
ppx_deriving < 5.0 does not use ppxlib

Reply:
but also 5.2.1 is compatible: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/5adf43152e79f493b22f5029fb5152e38cab1e9c/variant/opam-2.1,compilers,4.05,ppx_import.1.10.0,tests

Reply:
ppx_deriving is fine, but note you are not installing ppx_sexp_conv (and the dune file disable these tests)

I'm a bit lost on what you mean here.

Reply:
The change is already pushed upstream.

> I’m lost as well. CI shows everything working on 4.05 upwards so I’m merging. Thanks a lot!

It is working because you placed the or constraint, my comment was for the version without the or constraint, which can't possible work.

Thanks!

Reply:
In particular note this https://github.com/ocaml-ppx/ppx_import/blob/master/src_test/ppx_deriving_sexp/dune#L3